docs/controls/DockPanel.md

```diff
@@ -0,0 +1,73 @@
+---
+title: DockPanel XAML Control
+author: nmetulev
+ms.date: 14/10/2017
```

Comment:
Date format should be mm/dd/yyyy

Also, feel free to add your username as author